ansible/roles/aws_ecs/files/etc/init/ecs_agent.conf

```diff
@@ -8,7 +8,7 @@ respawn limit 100 1
 
 # Needed to find /root/.dockercfg
 env HOME="/root"
-env IMAGE=amazon/amazon-ecs-agent@sha256:54e98d053d0fd6735732e2c17054ab5d78b40be9d20a628fce993dc789240399 # v1.2.1 tag
+env IMAGE=amazon/amazon-ecs-agent@sha256:72c02d9a96340e4381c70b4355289e204b36e7e667352eed8df4d0e2f2d308a4 # v1.3.1 tag
```

Comment:
I'm still curious how yo get the sha.  What file do you look at to compute this?